packages/auto_mappr/lib/src/builder/methods/safe_convert_method_builder.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:auto_mappr/src/builder/methods/methods.dart';
+import 'package:auto_mappr/src/extensions/expression_extension.dart';
+import 'package:code_builder/code_builder.dart';
+
+// modules OK
+// modules tests OK
```

Comment:
remove comments (maybe they are somewhere else also, they were just dev notes and you can remove them too)